### DISCUSSION TITLE
Add option to manually select (and/or list) coin IDs when sending transaction in GUI, as well as transaction hash information

### DISCUSSION THREAD

**Discussion by OverActiveBladderSystem** [Chia-Network/chia-blockchain-gui#2373] [Ideas]
When sending a transaction in the GUI there may be times when manually selecting coin IDs (and seeing their value) is preferred to the current selection process, giving GUI users more control over spends.

![Manually select coins](https://github.com/Chia-Network/chia-blockchain/assets/36348012/6d0b3f2e-6773-4f4c-afb1-f41a43252fe0)

Side quest, there may be times someone wants to see a list of existing coin IDs (and their value) in the wallet or for a transaction when using the GUI, an option for this someplace on the wallets main page and/or as extended information on a transaction.

This could also tie in to recording and displaying the transaction hash for the user, for use with blockchain explorers or to send to another party.

![coin id list](https://github.com/Chia-Network/chia-blockchain/assets/36348012/892124cc-d45f-4fb9-95dc-f4a2946fcef7)

**Comment by BrandtH22** [comment]
@OverActiveBladderSystem , note I transferred this enhancement request to the GUI repo for better visibility 👍
